src/store/Disks.h

```diff
@@ -42,14 +42,19 @@ class Disks: public Controlled
     virtual void evictIfFound(const cache_key *) override;
     virtual int callback() override;
 
-    /// slowly calculate (and cache) hi/lo watermarks and similar limits
-    void updateLimits();
+    /// update configuration, including limits (re)calculation
+    void configure();
+    /// parses a single cache_dir configuration line
+    static void Parse(Store::DiskConfig &);
```

Comment:
If possible, please remove `Store::` inside `Store` in new/changed code. There are several occurrences.